.changeset/pr9-enhanced-layer-support.md

```diff
@@ -0,0 +1,14 @@
+---
+"@module-federation/enhanced": minor
```

Comment:
make this a patch and update it based on the actual changes in the pr (if any) otherwise delete